clmm/theory/parent_class.py

```diff
@@ -321,19 +322,53 @@ def eval_reduced_tangential_shear(self, r_proj, z_cl, z_src):
             Galaxy cluster redshift
         z_src : array_like, float
             Background source galaxy redshift(s)
+        z_src_model : str, optional
+            Source redshift model, with the following supported options:
+
+                * `single_plane` (default): all sources at one redshift (if `z_source` is a float) \
+                    or known individual source galaxy redshifts (if `z_source` is an array and \
+                    `r_proj` is a float);
+                * `weighing_the_giants_b`: use the equation (6) in Weighing the Giants - III \
+                    (https://arxiv.org/abs/1208.0605) to evaluate tangential reduced shear;
```

Comment:
Same comment as above regarding the name of that option

Reply:
Done.

Reply:
Done.